doc/index.rst

```diff
@@ -0,0 +1,93 @@
+#########################
+LinearStage
+#########################
+
+.. image:: https://img.shields.io/badge/SAL-API-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/LinearStage.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_LinearStage
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_LinearStage
+.. image:: https://img.shields.io/badge/Jenkins-gray.svg
+    :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_LinearStage/
+
+.. TODO: Delete the note when the page becomes populated
+
+.. Warning::
+
+   **This CSC documentation is under development and not ready for active use.**
+
+.. _index:overview:
+
+Overview
+========
+
+:ref:`Contact info <ts_xml:index:master-csc-table:LinearStage>`
+
+The LinearStage controls a Zaber single axis linear stage.
```

Comment:
add the link from the dev guide above? State that it's an LST model (since there are lots).